### DISCUSSION TITLE
How do I use a .env file to manage secrets?

### DISCUSSION THREAD

**Discussion by braveterry** [usebruno/bruno#1657] [Q&A] [answered]
I have looked [here](https://docs.usebruno.com/secrets-management/dotenv-file.html), but it doesn't show the format of the .env file.  It's not clear if there are constraints around how it is named either.

[This post](https://github.com/usebruno/bruno/discussions/828#discussioncomment-7427236) makes it seem like the .env file is a simple property file with key/value pairs.  Based on that, here is what I tried:

I created a "Test Collection".

In the root of the "Test Collection" directory, I created a `my.env` file that contains this single line:

    TEST_APIKEY = 123456789101112

I created a PROD environment and created an entry like this:

![Screenshot of my PROD environment config](https://github.com/usebruno/bruno/assets/527650/48fc0085-beae-49af-a147-cbabdcd20af6)

I am passing the `apikey` as a query parameter like this:

![image](https://github.com/usebruno/bruno/assets/527650/287213fd-55a8-428e-9f9d-3a4d3fd2cb1e)

When I call the service, here is what gets passed:

![image](https://github.com/usebruno/bruno/assets/527650/21ba3448-82e5-4faf-b731-5ff61983196d)

As you can see, it's not passing the value.

*What am I doing wrong?*

Thanks!

**Comment by sanjai0py** [comment, marked as answer]
Hey! How's it going, @braveterry? I noticed that you've done everything correctly, but it seems like you may have made a small mistake with your env file name. Just wanted to let you know that conventionally, environment variables should contain a leading `.` in the file name. So instead of `my.env`, your file should look something like `.env`. Thanks!

**Reply by braveterry** [reply]
That worked.  I thought I could name the file `*.env`, but it should literally be named `.env`.  🙂

Thanks!
